### ISSUE TITLE
Output scale differences in LSTM vs. HMM

### ISSUE THREAD

**Issue by LarsGab** [Gaius-Augustus/Tiberius#49]
The LSTM outputs probabilities, whereas the HMM outputs logarithmic scores. The loss function handles these scale differences by using the `from_logits` parameter and by converting one scale to the other. We should standardize the output scales to prevent unintended issues.
